apps/docs/src/app/constants/locale.ts

```diff
@@ -0,0 +1,8 @@
+export enum DocsLocale {
+    En = 'en',
+    Ru = 'ru'
+}
+
+export const DOCS_DEFAULT_LOCALE = DocsLocale.Ru;
+
+export const DOCS_SUPPORTED_LOCALES = Object.values(DocsLocale);
```

Comment:
можно при инициализации тип задать , чтобы в дальнейшем его не приводить к другому типу
```suggestion
export const DOCS_SUPPORTED_LOCALES = Object.values<string>(DocsLocale);
```